zerver/lib/typed_endpoint.py

```diff
@@ -316,10 +316,12 @@ def parse_view_func_signature(
     "datetime_parsing": _("{var_name} is not a date"),
     "datetime_type": _("{var_name} is not a date"),
     "dict_type": _("{var_name} is not a dict"),
+    "enum": _("Invalid {var_name}"),
```

Comment:
This keeps the error same as it was for the integer type settings before using `typed_endpoint`.